_posts/2023-05-31-mlops-platform-sandbox.md

```diff
@@ -0,0 +1,58 @@
+
+## How to Use the Sandbox
+
+The Sandbox provides you with pre-built pipelines that you can easily re-run or modify to suit your needs. If you want to run your own custom code inside your sandbox, there are a few more steps you'll have to do, all of which is [described in our dedicated documentation page](https://docs.zenml.io/user-guide/advanced-guide/sandbox).
+
+To list the available pre-built pipelines with which your sandbox comes, simply use the command `zenml pipeline list`. This will display a list of pipelines, including their build IDs, which you can use to run a specific pipeline.
+
+To run a pipeline with a build, use the command `zenml pipeline run <PIPELINE_NAME> --build_id <BUILD_ID>`. For example, to run the `langchain_pipeline`, you would enter `zenml pipeline run langchain_pipeline --build_id <BUILD_ID>`, replacing `<BUILD_ID>` with the appropriate build ID from the list.
+
+Using these pre-built pipelines makes it incredibly easy to reproduce results and experiment with powerful tools integrated into the ZenML framework. By leveraging the MLOps Platform Sandbox, you can quickly explore the capabilities of a unified MLOps platform in the context of real ML pipelines without the hassle of setting up and managing your own infrastructure.
+
+## What To Do After Your Sandbox Expires
+
+Once you have experimented with the MLOps Platform Sandbox and gained a better understanding of how the ZenML frameworks work, you may want to deploy your own MLOps stack tailored to your specific needs. To help you with this process, ZenML offers [Stack Recipes](https://github.com/zenml-io/mlops-stacks), which provide a starting point for deploying various MLOps stacks on different cloud providers and with different components.
+
+[Stack Recipes](https://github.com/zenml-io/mlops-stacks) are designed to be customizable, allowing you to easily swap out components and infrastructure based on your company's requirements. You can replace Kubeflow with Vertex AI Pipelines or Sagemaker Pipelines, use S3 or GCS storage instead of Minio, and choose Weights and Biases or Neptune over MLflow.
+
+To get started with deploying your own MLOps stack, visit the [ZenML Stack Recipes](https://github.com/zenml-io/mlops-stacks) repository along with [our documentation page](https://docs.zenml.io/platform-guide/set-up-your-mlops-platform/deploy-and-set-up-a-cloud-stack/deploy-a-stack-post-sandbox) and follow the instructions provided for your desired stack configuration. This will enable you to build a robust, production-ready MLOps platform tailored to your specific needs, while leveraging the knowledge and experience gained from using the MLOps Platform Sandbox.
```

Comment:
Maybe add a CTA to the form to extend the sandbox which is: https://zenml.io/extend-sandbox.

Reply:
that link gives me a 404. Also what form are you talking about?

Reply:
This link will be fixed soon, its a form that lets people ask for an extended mlops platform sandbox

Reply:
got it. I'll update the relevant part of docs as well? and the FAQ?

Reply:
@bcdurak already added a card in the docs that I wrote, for the FAQ ill do it on the website directly